components/forms/TableBulkEditPopover/style.module.scss

```diff
@@ -0,0 +1,35 @@
+@import "styles/index";
+
+.popover-wrapper {
+  position: absolute;
+
+  &--top {
+    top: 38%;
+  }
+  
+  &--left {
+    left : 5%;
+  }
+
+  &--position-left {
+    right: 100%;
+    position: absolute;
+    background: $color-night;
+    width: 54px;
+    height: 43px;
+  }
+
+  &--position-top {
+    bottom: 100%;
+    position: absolute;
+    background: $color-night;
+    width: 54px;
+    height: 43px;
+  }
+
+  &--cell {
```

Comment:
Also, no pixel measurements. Just the border. The table that will make use of this will take care of the padding/sizes. We just need to center the cell and give it a border color.